spec/lib/driver_commission_history_response_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'spec_helper'
+require 'gman_client'
+require 'vcr'
+require 'httparty'
+require 'webmock'
+require 'support/vcr'
+
+RSpec.describe GmanClient do
+  describe '.receive_driver_commission_history_response' do
+    VCR.use_cassette('driver_commission_history') do
+      response = GmanClient.driver_commission_histories
+      subject(:driver_response) { response }
+      let(:driver_commission) { driver_response.first }
+
+      it 'empty is not false' do
+        expect(driver_response.empty?).to be_falsey
```

Comment:
`it 'empty is not false' do` but it is `expect(driver_response.empty?).to be_falsey`